install_requirements.py

```diff
@@ -1,19 +1,20 @@
-import pip
+import pip._internal
 import os
```

Comment:
we'll need to discuss this in person and determine the actual issue, this seems like something that is dependent on the version of pip.